Template/User_HTML/formpage.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>twitterclone_signup</title>
+  </head>
+
+  <body>
+    <h1>Twitter_cloneapp!</h1>
+    <h2>登録画面</h2>
+    <form action="" method="post">
+      {% csrf_token %}
+      {{ form.as_p }}
+      <input class="btn" type="submit"  value="登録">
+    </form>
+  </body>
+
+</html>
```

Comment:
全てのファイルにおいて、最後は1行空行を挟むようにしましょう！
github上で`no new line at end of file `というエラーが起こっています。
[行末の空白削除と改行は何故しなきゃいけないの？](https://bottoms-programming.com/archives/indent-tab-space-and-new-line.html#toc5)
```suggestion
</html>

```